jobs/e2e_node/containerd/k8s-release-branches/image-config-1.30.yaml

```diff
@@ -1,5 +1,5 @@
 images:
-  cos-101:
+  cos-dev:
     image_family: cos-109-lts
```

Comment:
@ndixita should we switch to 113 for 1.30? It may be more aligned with the support window

I'm going to approve this PR, please send a separate one to catch up on the COS family